### PR TITLE
Update noVNC to 1.4.0

### PR DESCRIPTION
**Pull request by AntonAndreevichMoroz**
Updated noVNC to version 1.4.0. Necessary changes have been made to work in MeshCentral. Checked locally, ZRLE support works.

### CODE REVIEW

File: public/novnc/vnc.html [hunk 1]
```diff
@@ -15,53 +15,35 @@
     -->
     <title>noVNC</title>
 
-    <meta charset="utf-8">
```
Comment (si458) [inline public/novnc/vnc.html:18]: why have you removed all the icons and also the meta header? isnt it going to stop utf-8 stuff working?

Comment (si458) [reply public/novnc/vnc.html:18]: ok mine also removes the utf-8 for some reason?
but why have you removed all the icons?
these are needed to show pretty icons on menus etc

**Comment by si458** [comment]
all looks good to me! nice catch on the few lines that my PR removed and shouldnt of! 
thats what i get for copying and pasting!

**Comment by Ylianst** [comment]
This is wonderful!! I will release the next version with this included!

**Comment by si458** [comment]
Fixes #4928

**Pull request merged**